source/vibe/data/json.d

```diff
@@ -181,6 +199,8 @@ struct Json {
 	/// ditto
 	long opAssign(long v) { m_type = Type.int_; m_int = v; return v; }
 	/// ditto
+	BigInt opAssign(BigInt v) { m_type = Type.bigint; initBigInt(); m_bigint = v; return v; }
```

Comment:
What I was refering to was, if someone calls `opAssign` with a `BigInt` value multiple time, the finalizers of all but the last 'instance' will not be run.